scipy/signal/signaltools.py

```diff
@@ -1786,8 +1786,8 @@ def resample(x, num, t=None, axis=0, window=None):
 
     See also
     --------
-    decimate
-    resample_poly
+    decimate : Downsample the signal after applying an anti-aliasing filter.
```

Comment:
it's a bit weird to mention the anti-aliasing filter here but not for `resample` where it's also implicitly done by truncation or `resample_poly` where it's done explicitly